tests/test_ckan_to_frictionless.py

```diff
@@ -264,6 +264,56 @@ def test_dataset_extras(self):
             'location': {'country': 'China'},
         }
 
+    def test_unjsonify_all_extra_values_in_nested_dicts(self):
+        self.dataset_dict.update({
+            'extras': [
+                {
+                    'key': 'location',
+                    'value': ('{"country": {"China": {"population": '
+                              '"1233214331", "capital": "Beijing"}}}')
+                }
+            ]
+        })
+        out = converter.dataset(self.dataset_dict)
+        exp = {'location':
+               {'country':
+                {'China': {'population': '1233214331',
+                           'capital': 'Beijing'}}
+                }
+               }
+        assert out.get('extras') == exp
+
+    def test_unjsonify_all_extra_values_in_nested_lists(self):
```

Comment:
I get the value of separate tests here but i think you could probably just have one and multiple entries in extras dict. It would be simpler and easier to follow i think.

Reply:
Yes, I think there is overlap anyways as once one of those tests passed, the others did too. I can certainly refactor that into a single test with three separate keys instead of three separate tests if you would like that best. It would make the resulting dictionary a bit more complex and realistic and would still test everything in one go. :+1: